Update export to be at first level

thibmeu opened the pull request:
Emports were grouped based on internal file representation. This makes them inconvenient to import. For instance, to import `PrivateToken`, one would need to
```typescript
import { httpAuthScheme } from "@cloudflare/privacypass-ts";

httpAuthScheme.PrivateToken
```
This also makes interaction between methods hard to follow. To create such a token, you need to
```typescript
import { issuance } from "@cloudflare/privacypass-ts";

issuance.createPrivateToken(...)
```

This commit moves methods closer to their respective types, and exports types at the first level.
One can now
```typescript
import { PrivateToken } from "@cloudflare/privacypass-ts";

PrivateToken.fromInfos(...)
```
This commit also renames `TokenType` to `TOKENS_TYPES.BLIND_RSA`.

armfazh commented on examples/pubVerifiable.example.ts:
```diff
@@ -37,7 +37,12 @@ export async function publicVerifiableTokens(): Promise<void> {
     const redemptionContext = crypto.getRandomValues(new Uint8Array(32));
     const originInfo = ['origin.example.com', 'origin2.example.com'];
-    const tokChl = await createPrivateToken(issuer, redemptionContext, originInfo);
+    const tokChl = await PrivateToken.fromInfos(
```
`PrivateToken.create` is a more natural name.

thibmeu replied:
updated

armfazh reviewed (approved):
just a comment about `create` method

thibmeu committed:
Update export to be at first level

Emports were grouped based on internal file representation. This makes
them inconvenient to import. For instance, to import `PrivateToken`, one
would need to `import { httpAuthScheme } from
"@cloudflare/privacypass-ts"; httpAuthScheme.PrivateToken`.
This also makes interaction between methods hard to follow. To create
such a token, you need to `import { issuance } from "@cloudflare/privacypass-ts"; issuance.createPrivateToken(...)`.

This commit moves methods closer to their respective types, and exports
types at the first level.
One can now `import { PrivateToken } from "@cloudflare/privacypass-ts";
PrivateToken.fromInfos(...)`.
This commit also renames `TokenType` to `TOKENS_TYPES.BLIND_RSA`.
